packages/pillarbox-playlist/test/pillarbox-playlist-ui.spec.js

```diff
@@ -4,52 +4,23 @@ import '../src/pillarbox-playlist.js';
 import PillarboxPlaylistUi from '../src/pillarbox-playlist-ui.js';
 import { RepeatMode } from '../src/pillarbox-playlist.js';
 
-const playlist = [
-  {
-    sources: [{
-      src: 'first-source',
-      type: 'test'
-    }],
-    poster: 'first-poster',
-    data: {
-      title: 'first-source',
-      duration: 180
-    }
-  },
-  {
-    sources: [{
-      src: 'second-source',
-      type: 'test'
-    }],
-    poster: 'second-poster',
-    data: {
-      title: 'second-source',
-      duration: 150
-    }
-  },
-  {
-    sources: [{
-      src: 'third-source',
-      type: 'test'
-    }],
-    poster: 'third-poster',
-    data: {
-      title: 'third-source',
-      duration: 120
-    }
-  },
-  {
-    sources: [{
-      src: 'fourth-source',
-      type: 'test'
-    }],
-    poster: 'fourth-poster',
-    data: {
-      title: 'fourth-source',
-      duration: 210
-    }
-  }
-];
+const playlist = [{
+  sources: [{ src: 'first-source', type: 'test' }],
+  poster: 'first-poster',
+  data: { title: 'first-source', duration: 180 }
+}, {
+  sources: [{ src: 'second-source', type: 'test' }],
+  poster: 'second-poster',
+  data: { title: 'second-source', duration: 150 }
+}, {
+  sources: [{ src: 'third-source', type: 'test' }],
+  poster: 'third-poster',
+  data: { title: 'third-source', duration: 120 }
+}, {
+  sources: [{ src: 'fourth-source', type: 'test' }],
+  poster: 'fourth-poster',
+  data: { title: 'fourth-source', duration: 210 }
+}];
```

Comment:
Do we really need to change the indentation here when nothing has changed?  If the IDE did this on its own, it would be useful to use a tool such as **prettier** or **biome** so that the indentation does not change according to the whims of the IDE or the developer.